docs/general/administration/hardware-selection.md

```diff
@@ -0,0 +1,158 @@
+---
+uid: admin-hardware-selection
+title: Hardware Selection
+---
+
+# Selecting Appropriate Hardware
+
+This page covers what you need to know in order to select appropriate hardware for a Jellyfin server and take full advantage of its features (e. g.. hardware acceleration).
+
+## TL;DR
+
+For a Jellyfin server, the following is recommended:
+
+### Normal Server From Scratch
+
+- CPU: 4 threads
+  - Please Avoid 11th gen or older Intel Atom and Celeron / Pentium N / J series CPUs.
+  - Please avoid Rapsberry PIs and other low powered SBCs.
+  - Please avoid Intel Core M / Y series CPUs.
+- RAM: 8GB or more
+- Storage: SSD for Jellyfin files and transcoding cache.
+- Graphics: Intel > Nvidia > AMD > Apple.
+  - A system with Intel 7th gen or newer Core series CPU with integrated graphics is recommended.
+  - When using dedicated graphics, please consider using Intel ARC over Nvidia graphics.
+  - When using Nvidia graphics, GTX 16 / RTX 20 (Turing) or newer is recommended, excluding GTX 1650.
+  - AMD Graphics or Apple Silicon is NOT recommended due to poor H.264 and H.265(HEVC) encoder quality.
+
+:::note A Note on Apple hardware
+
+It is recommended that Intel-based Macs be used with Windows or Linux installed to host Jellyfin. Many hardware acceleration features aren't available on MacOS due to the custom [jellyfin-ffmpeg](https://github.com/jellyfin/jellyfin-ffmpeg) fork not being available.
+
+:::
+
+### Low Power Applications
+
+For low power applications, Intel 12th gen or newer Atom CPUs with integrated graphics are recommended.
```

Comment:
Link to the LowPower settings would be nice

Reply:
Where is this located in the docs?

Reply:
https://jellyfin.org/docs/general/administration/hardware-acceleration/intel#low-power-encoding

Reply:
Linked